train.py

```diff
@@ -25,14 +23,20 @@
 num_features = context_embeddings.shape[2]
 raw_question = questions[0][0]['question']
 
-words_in_context = tf.placeholder(tf.float32, [time_steps, batch_size, num_features])
-lstm = tf.keras.layers.LSTM(units=3)
```

Comment:
From the doc: Note that this cell is not optimized for performance on GPU. Please use tf.compat.v1.keras.layers.CuDNNLSTM for better performance on GPU.
 But I think tf.keras.layers.CuDNNLSTM (https://www.tensorflow.org/versions/r1.14/api_docs/python/tf/keras/layers/CuDNNLSTM) can be used.

Reply:
oh I did not know this, good catch!

Reply:
This will not work on CPU, so we will keep it as is for debugging and switch over when training on a GPU, maybe add a flag?